<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SysSetMainDialog</class>
 <widget class="QDialog" name="SysSetMainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="CommonPushButton" name="pushButtonSysSetMainLag">
   <property name="geometry">
    <rect>
     <x>777</x>
     <y>86</y>
     <width>220</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>         语 言</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonSysSetMainPwd">
   <property name="geometry">
    <rect>
     <x>777</x>
     <y>159</y>
     <width>220</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>         密 码</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonSysSetMainTime">
   <property name="geometry">
    <rect>
     <x>777</x>
     <y>232</y>
     <width>220</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>         时 间</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonSysSetMainSecurity">
   <property name="geometry">
    <rect>
     <x>777</x>
     <y>451</y>
     <width>220</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>         安 防</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonSysSetMainDevice">
   <property name="geometry">
    <rect>
     <x>777</x>
     <y>378</y>
     <width>220</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>         设 备</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonSysSetMainRing">
   <property name="geometry">
    <rect>
     <x>777</x>
     <y>305</y>
     <width>220</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>         铃 声</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonSysSetMainVersion">
   <property name="geometry">
    <rect>
     <x>777</x>
     <y>13</y>
     <width>220</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>         版 本</string>
   </property>
  </widget>
  <widget class="QFrame" name="framePanel">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>12</y>
     <width>700</width>
     <height>580</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonSysSetMainReturn">
   <property name="geometry">
    <rect>
     <x>777</x>
     <y>524</y>
     <width>220</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string> 返回</string>
   </property>
  </widget>
  <zorder>framePanel</zorder>
  <zorder>pushButtonSysSetMainLag</zorder>
  <zorder>pushButtonSysSetMainPwd</zorder>
  <zorder>pushButtonSysSetMainTime</zorder>
  <zorder>pushButtonSysSetMainSecurity</zorder>
  <zorder>pushButtonSysSetMainDevice</zorder>
  <zorder>pushButtonSysSetMainRing</zorder>
  <zorder>pushButtonSysSetMainVersion</zorder>
  <zorder>pushButtonSysSetMainReturn</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CommonPushButton</class>
   <extends>QPushButton</extends>
   <header>commonpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButtonSysSetMainVersion</tabstop>
  <tabstop>pushButtonSysSetMainLag</tabstop>
  <tabstop>pushButtonSysSetMainPwd</tabstop>
  <tabstop>pushButtonSysSetMainTime</tabstop>
  <tabstop>pushButtonSysSetMainRing</tabstop>
  <tabstop>pushButtonSysSetMainDevice</tabstop>
  <tabstop>pushButtonSysSetMainSecurity</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
